### DISCUSSION TITLE
Please implement Kagi in Web Search options

### DISCUSSION THREAD

**Discussion by ylluminate** [open-webui/open-webui#7688] [General]
I would like to see Kagi added as a search provider for the Web Search functionality:
![image](https://p153.p0.n0.cdn.zight.com/items/12udqqZ2/7d0ddec9-959c-40cc-8a14-b075d2d8f9ae.png?v=c7481e130c0e06040f05fbeb683be6f3)

**Comment by tjbck** [comment]
PR Welcome!

**Reply by erics118** [reply]
I'm working on a PR https://github.com/open-webui/open-webui/pull/7694
